plugins/woocommerce-admin/client/two-column-tasks/task-list-completion-slot/utils.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { Slot, Fill } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { createOrderedChildren, sortFillsByOrder } from '../../utils';
+
+export const WC_TASK_LIST_COMPLETION_SLOT_NAME =
```

Comment:
The name is already quite mouthful, but I think we should add `Experimental` somewhere 😬

Reply:
ok 😆

Reply:
I don't really want to break things when we cut over, so I'll just change the variable name and leave the text string alone